skaffold.yaml

```diff
@@ -12,12 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: skaffold/v2alpha4
+apiVersion: skaffold/v2beta18
 kind: Config
+metadata:
+  name: setup # module defining setup steps
+deploy:
+  kubectl:
+    manifests: 
+    - extras/jwt/jwt-secret.yaml
+    - dev-kubernetes-manifests/config.yaml
+
+---
+
+apiVersion: skaffold/v2beta15
```

Comment:
Is there a reason we use `skaffold/v2beta15` instead of `skaffold/v2beta18`?

Reply:
fixed